Framework/Algorithms/src/CombineDiffCal.cpp

```diff
@@ -70,9 +70,60 @@ API::ITableWorkspace_sptr CombineDiffCal::sortTableWorkspace(DataObjects::TableW
   return alg->getProperty("OutputWorkspace");
 }
 
+bool findColumn(const std::vector<std::string> columnNames, const std::string name) {
+  return std::find(columnNames.begin(), columnNames.end(), name) != columnNames.end();
+}
+
+int hasAllExpectedDifcColumns(const DataObjects::TableWorkspace_sptr ws) {
```

Comment:
I think if I invert the 0's and 1's that would make the return value for the function more intuitive, that way it can still be used like a bool, worth changing?

Reply:
I've gotta fix a cppcheck failure anyway so I think I will

Reply:
Could you just try running [`TableWorkspace::getColumn`](https://github.com/mantidproject/mantid/blob/1962b53c85db77ac1c91c24093521d3a881fd9ad/Framework/DataObjects/src/TableWorkspace.cpp#L105) and catch the RuntimeError to see if the column exists?